modular_skyrat/modules/events/code/event_frequency.dm

```diff
@@ -0,0 +1,282 @@
+#define VERY_HIGH_EVENT_FREQ 64
+#define HIGH_EVENT_FREQ 32
+#define MED_EVENT_FREQ 16
+#define LOW_EVENT_FREQ 8
+#define VERY_LOW_EVENT_FREQ 4
+#define MIN_EVENT_FREQ 2
+
+/**
+ * This file is used to denote event weight and occurence.
+ */
+
+/**
+ * Event subsystem
+ *
+ * Overriden min and max start times:
+ * To accomodate for much longer rounds.
+ */
+/datum/controller/subsystem/events
+	frequency_lower = 15 MINUTES
+	frequency_upper = 25 MINUTES
+
+/datum/round_event_control/abductor
+	weight = VERY_LOW_EVENT_FREQ
+
+// Disabled: Part of dynamic.
+/datum/round_event_control/alien_infestation
+	max_occurrences = 0
+
+/datum/round_event_control/anomaly/anomaly_flux
```

Comment:
aren't a lot of these set by default?

Reply:
Yes, but for organization purposes this is easier to manage/edit as events get added and removed over time. Particularly added.